code/game/objects/structures/girders.dm

```diff
@@ -4,11 +4,25 @@
 	density = 1
 	var/state = 0
 	var/material = /obj/item/stack/sheet/metal
+	var/construction_length = 40
```

Comment:
Can't even choose your own variable names anymore in this Orwellian newspeak world smh